guides/server/live-layouts.md

```diff
@@ -48,6 +48,30 @@ ways for flexibility:
   1. The `:layout` option in `Phoenix.LiveView.Router.live_session/2`,
      when set, will override the `:layout` option given via
      `use Phoenix.LiveView`
+     
+    scope "/", MyappWeb do
+      pipe_through [:browser, :redirect_if_user_is_authenticated]
+  
+      live_session :redirect_if_user_is_authenticated,
+        on_mount: [{MyappWeb.UserAuth, :redirect_if_user_is_authenticated}],
+        layout: {MyappWeb.Layouts, :main_app} do
+        live "/users/register", UserRegistrationLive, :new
+        live "/users/log_in", UserLoginLive, :new
+        live "/users/reset_password", UserForgotPasswordLive, :new
+        live "/users/reset_password/:token", UserResetPasswordLive, :edit
+      end
+  
+      post "/users/log_in", UserSessionController, :create
+    end
+
+  Folder layout:
+
+    myapp_web/
+      -> components
+         -> layouts
+            -> main_app.html.heex
+            
+   main_app.html.heex should `<%= @inner_content %>`
```

Comment:
Can you please reduce the example and make sure that it renders nicely with `MIX_ENV=docs mix docs`? We also mention that it must use `<%= @inner_content %>` earlier, so I think we can remove it from here. :)